gradle.properties

```diff
@@ -1,6 +1,6 @@
 TILEDB_GIT_REPOSITORY=https://github.com/TileDB-Inc/TileDB
 TILEDB_GIT_TAG=2.0.3
```

Comment:
Should this be bumped to 2.0.8 while you're at it?

Reply:
I was going to do that in a separate PR incase these changes for serialization/s3 need to be backed out.